run/test/specs/disappearing_message_checks_ios.spec.ts

```diff
@@ -0,0 +1,788 @@
+    strategy: "accessibility id",
+    selector: "Message body",
+    maxWait: 1000,
+    text: testLink,
+  });
+  await device2.hasElementBeenDeletedNew({
+    strategy: "accessibility id",
+    selector: "Message body",
+    maxWait: 1000,
+    text: testLink,
+  });
+  await closeApp(device1, device2);
+}
+
+async function disappearingCommunityInviteMessage1o1(
+  platform: SupportedPlatformsType
+) {
+  const { device1, device2 } = await openAppTwoDevices(platform);
+  const communityLink = `https://chat.lokinet.dev/testing-all-the-things?public_key=1d7e7f92b1ed3643855c98ecac02fc7274033a3467653f047d6e433540c03f17`;
+  const communityName = "Testing All The Things!";
+  // Create user A and user B
+  const [userA, userB] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+  ]);
+  await newContact(platform, device1, userA, device2, userB);
+  await setDisappearingMessage(device1, [
+    "1o1",
+    "Disappear after send option",
+    "10 seconds",
+  ]);
+  await device1.navigateBack(platform);
+  await device1.navigateBack(platform);
+  await device1.clickOnElement("New conversation button");
+  await device1.clickOnElement("Join Community");
+  await device1.inputText(
+    "accessibility id",
+    "Enter Community URL",
+    communityLink
+  );
+  await device1.clickOnElement("Join");
+  // Wait for community to load
+  await sleepFor(1000);
+
+  await device1.clickOnElement("More options");
+  await device1.clickOnElement("Add Members");
+  await device1.clickOnElementByText({
+    strategy: "accessibility id",
+    selector: "Contact",
+    text: userB.userName,
+  });
+  await device1.clickOnElement("Done");
+  // Check device 2 for invitation from user A
+  await device2.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Community invitation",
+    text: communityName,
+  });
+  // Wait for 10 seconds for message to disappear
+  await sleepFor(10000);
+  await device2.hasElementBeenDeletedNew({
+    strategy: "accessibility id",
+    selector: "Message body",
+    maxWait: 1000,
+    text: communityName,
+  });
+  await device1.hasElementBeenDeletedNew({
+    strategy: "accessibility id",
+    selector: "Message body",
+    maxWait: 1000,
+    text: communityName,
+  });
+  await closeApp(device1, device2);
+}
+
+async function disappearingCallMessage1o1(platform: SupportedPlatformsType) {
+  const { device1, device2 } = await openAppTwoDevices(platform);
+  // Create user A and user B
+  const [userA, userB] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+  ]);
+  await newContact(platform, device1, userA, device2, userB);
+  await setDisappearingMessage(device1, [
+    "1o1",
+    "Disappear after send option",
+    "10 seconds",
+  ]);
+  await device1.navigateBack(platform);
+  await device1.clickOnElement("Call");
+  // Enabled voice calls in privacy settings
```

Comment:
you have to write some shared function for those tests, there is too much duplication